atomic_reactor/plugins/post_pulp_pull.py

```diff
@@ -28,7 +28,11 @@ class CraneTimeoutError(Exception):
     pass
 
 
-class PulpPullPlugin(PostBuildPlugin):
+# Note: We use multiple inheritance here only to make it explicit that
```

Comment:
This comment contradicts the commit message in the sense that it states it's only done for clarity. 
The commit message makes it seem that this is actually required for functionality.